docs-partials/builder/azure/arm/SharedImageGallery-not-required.mdx

```diff
@@ -14,4 +14,8 @@
   version across regions set this value to one that is available in all
   regions where you are deploying.
 
+- `communityGallery_image_id` (string) - Id of the community gallery image : /CommunityGalleries/{cgName}/Images/{img}[/Versions/{}] (Versions part is optional)
```

Comment:
Ditto commit.

Reply:
Sorry I don't understand, is there an issue?

Reply:
aah put it in the commit message? added!